source/assetlist.json

```diff
@@ -0,0 +1,34 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "source",
+  "assets": [
+    {
+      "description": "The native token of SOURCE Chain",
+      "denom_units": [
+        {
+          "denom": "usource",
+          "exponent": 0
+        },
+        {
+          "denom": "source",
+          "exponent": 6
+        }
+      ],
+      "base": "usource",
+      "name": "Source",
+      "display": "source",
+      "symbol": "SOURCE",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/source/images/source.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/source/images/source.svg"
+      },
+      "coingecko_id": "source-protocol",
```

Comment:
Is this the same token? Feels like it could be different
https://www.coingecko.com/en/coins/source-protocol